### DISCUSSION TITLE
route voice prompts enquiry

### DISCUSSION THREAD

**Discussion by ormerode** [osmandapp/OsmAnd#12997] [General]
I’m brand new to OSmAand. Its quite good. I have a question. 

I imported a GPX map to follow a mountain bike trail. The voice prompt was going all the time telling me to go slightly left, slightly right on a forest trail bends There were jus too many prompts. I just need the prompts for the main forks and crossroads 

Can I set up the voice prompts to only prompt where I need to turn onto another track, trail or road?  

Thanks

**Comment by vshcherb** [comment]
You can use 'Atttach to roads '  feature it will connect roads to OSM and voice helper will understand roads graph connectivity.
